chain/client/src/stateless_validation/validate.rs

```diff
@@ -14,11 +14,30 @@ use near_primitives::stateless_validation::partial_witness::{
 use near_primitives::types::{AccountId, BlockHeightDelta};
 use near_primitives::validator_signer::ValidatorSigner;
 use near_store::{DBCol, FINAL_HEAD_KEY, HEAD_KEY, Store};
+use strum::IntoStaticStr;
 
 /// This is taken to be the same value as near_chunks::chunk_cache::MAX_HEIGHTS_AHEAD, and we
 /// reject partial witnesses with height more than this value above the height of our current HEAD
 const MAX_HEIGHTS_AHEAD: BlockHeightDelta = 5;
 
+#[derive(IntoStaticStr)]
+#[strum(serialize_all = "snake_case")]
+pub enum ChunkRelevance {
```

Comment:
Should that be ChunkEndorsementRelevance? 

nit: Can you add a short comment and mention that all variants, except for Relevant, will be discarded as per is_relevant?

Reply:
This is also used for partial witness validation. Correct me if I'm wrong, but I believe that use case is not directly related to chunk endorsement.